bucket/altdrag.json

```
@@ -0,0 +1,20 @@
+{
+    "homepage": "https://github.com/stefansundin/altdrag",
```

io homepage would be better

```suggestion
    "homepage": "https://stefansundin.github.io/altdrag/",
```

---

Change checkver after that